app/views/layouts/landing_page_layout.html.erb

```diff
@@ -55,6 +55,23 @@
             Help for early years providers
           </a>
         </div>
+        <div>
+          <div class="gem-c-emergency-banner" aria-label="emergency banner" role= "region" data-nosnippet="true">
+            <div class="govuk-width-container">
+              <div class="govuk-grid-row">
+                <div class="govuk-grid-column-two-thirds">
+                  <h2 class="gem-c-emergency-banner__heading">Her Majesty Queen Elizabeth II</h2>
+                  <p class="gem-c-emergency-banner__description">
+                    21 April 1926 to 8 September 2022
+                  </p>
+                  <a href="" class="gem-c-emergency-banner__link">
```

Comment:
`href` is empty. Should it have a link in it?

Reply:
woops, yes it should